nilearn/plotting/tests/test_baseline_comparisons.py

```diff
@@ -168,6 +168,9 @@ def test_plot_carpet_default_params(img_4d_mni, img_3d_ones_mni):
 @pytest.mark.mpl_image_compare
 def test_plot_prob_atlas_default_params(img_3d_mni, img_4d_mni):
     """Smoke-test for plot_prob_atlas with default arguments."""
+    # TODO for 0.13.0
```

Comment:
Just to make sure I understand this to-do, why is this for 0.13.0 rather than 0.12.1 ?

Reply:
We only compare the figure generated by the tests in that file to the baseline figure  when testing with python 3.9 and the oldest supported version of our dependencies 
So we must generate the baseline figures with the same version of the dependencies: but I cannot generate the figures because of https://github.com/nilearn/nilearn/pull/5531#discussion_r2195532885

This should not be an issue in version 0.13 when move to matplotlib 3.8 (ish).